onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -198,6 +198,40 @@ void CUDAExecutionProvider::PerThreadContext::IncrementRegularRunCountBeforeGrap
 }
 #endif
 
+std::optional<std::string> LoadEnv(const std::string& name, const std::unordered_set<std::string>& valid_values) {
```

Comment:
Is this code duplicated for rocm ep as well?

Reply:
Moved to `onnxruntime/core/platform/env_var_utils.h`